src/cpp/session/modules/SessionTests.R

```diff
@@ -41,3 +41,7 @@
 
    .rs.scalar(result)
 })
+
+.rs.addJsonRpcHandler("check_test_file_exists", function(path) {
```

Comment:
You don't need a new RPC here -- elsewhere we test for file existence using `FilesServerOperations::stat`. Example:

https://github.com/rstudio/rstudio/blob/770b2b56ba60f87b28fd69f5accd49bfeb005368/src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/TextEditingTargetRMarkdownHelper.java#L633-L639

Reply:
Thanks! Reusing RPC now.